### PR TITLE
Add ability to set alarm from command

### PR DESCRIPTION
**Pull request by ChatPion**
Add optional parameter `given_time` in pylarm to support alarm from CLI.

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -15,14 +15,18 @@
...
         pylarm(src_path)
         
     elif argv_len == 2:
         try:
-            print(arguments[ '2' ][ sys.argv[1] ][ 'info' ])
+            # passed argument is in cli_options.json
+            if sys.argv[ 1 ] in arguments[ '2' ]:
+                print(arguments[ '2' ][ sys.argv[1] ][ 'info' ])
+            else:
+                pylarm(src_path, given_time=sys.argv[1])
```
Comment (bbenefield89) [inline src/main.py]: This line right here allows options such as `foo` or `--foo` for example pass through and are attempted to become parsed as a time in `pylarm.py`

What originally would happen if you were to put in an invalid option is fall to one of the `excepts` and print our an error message called from `print_cli_options_error(sys.argv)`.

**Comment by bbenefield89** [comment]
Hi @ChatPion thanks for taking an interest in **Pylarm**.

A lot of this looks great but for consistency reasons, I would prefer if users were able to set their alarm without having to specify a meridiem.

You are more than welcome to add this functionality to include a meridiem but by default I would prefer it without.

**Comment by ChatPion** [comment]
In the original issue (#19) an example with a meridien was added, but I can remove it if you want.
I'll try a fix to only evaluate times

**Comment by ChatPion** [comment]
I don't know if I should check whether the input is a valid hour in 12 hours format or that should be done in pylarm.py

**Comment by bbenefield89** [comment]
@ChatPion thanks for the awesome work! We couldn't have done it without you.

As soon as I can, I will add you to the contributors list.

If you would like to see your name up there even faster though, you are free to make a new PR from a different branch that adds you to the list in the README.md.

**If you decide to make another PR just be sure to use the PR template this time.**

**Pull request merged**